### PR TITLE
#1925 Provide a simple and better description in the README.md

### PR DESCRIPTION
**Pull request by vitojeng**
required by #1925 

### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by vitojeng** [comment]
@oharastream/developer,
Provide a simple and better description in the README.md :)

File: README.md [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
+
+Ohara provide an easy way to help you building a big-data streaming process environment based on [Apache Kafka](https://kafka.apache.org/):
+
+* Super simple way to create following clusters:
+  * [Apache Zookeeper](https://zookeeper.apache.org/) cluster
+  * Apache Kafka - Broker cluster
+  * Apache Kafka - Connect worker cluster
+* Connect various storage: Build-in ready to use connectors for popular storage.
+* Pipeline graph: A great user interface for your data pipeline design.
+* Stream-app: Develop your own stream-processing logic for data transformation.
```
Comment (chia7712) [inline README.md]: stream-app or streamapp?

File: README.md [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
 [![Build Status](https://builds.is-land.com.tw/buildStatus/icon?job=PostCommit-OHARA)](https://builds.is-land.com.tw/job/PostCommit-OHARA/)
 [![Documentation Status](https://readthedocs.org/projects/ohara/badge/?version=latest)](https://ohara.readthedocs.io/en/latest/?badge=latest)
+[![Github Assets](https://img.shields.io/badge/download-Assets-brightgreen.svg)](https://github.com/oharastream/ohara/releases)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![jFrog Bintray](https://img.shields.io/bintray/v/oharastream/ohara/ohara-client.svg)](https://bintray.com/oharastream/ohara)
 
-----------
+# Ohara
+
+Ohara provide an easy way to help you building a big-data streaming process environment based on [Apache Kafka](https://kafka.apache.org/):
```
Comment (chia7712) [inline README.md]: provides

Comment (chia7712) [inline README.md]: 我偏好的重點有二：
1. ohara是一個基於kafka和docker的大數據平台
2. ohara提供一個方便快速的方式建構streaming application

英文的部份例如：
Ohara, which is a big data platform powered by Apache Kafka and Docker, enables effective and quickly ways of building application at streaming and scale.

File: README.md (hunk 1 above)
Comment (vitojeng) [reply README.md]: 用 StreamApp 好了

File: README.md (hunk 2 above)
Comment (vitojeng) [reply README.md]: 好喔, 感謝 ~

File: README.md [hunk 3]
```diff
@@ -1,37 +1,43 @@
...
+
+#### [Documentation](https://ohara.readthedocs.io/en/latest)
 
-## Documentation
 
-Please goto following link to read document: [Latest document](https://ohara.readthedocs.io/en/latest)
+## Contributing
 
-We use [Read the Docs](https://readthedocs.org/) for automating building and hosting.
+We are working hard to create a great product, if you are interesting, any contribution is very welcome.
```
Comment (konekoya) [inline README.md]: My two cents :)

Ohara is a completely open source project. If you're interested in contributing, you can start from the [contrubting guide](https://ohara.readthedocs.io/en/latest/contributing.html). Also, take a look at our [issue tracker](https://github.com/oharastream/ohara/issues), the [good first issues](https://github.com/oharastream/ohara/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) would be a good place to start

File: README.md [hunk 4]
```diff
@@ -1,37 +1,43 @@
...
 [![Documentation Status](https://readthedocs.org/projects/ohara/badge/?version=latest)](https://ohara.readthedocs.io/en/latest/?badge=latest)
+[![Github Assets](https://img.shields.io/badge/download-assets-brightgreen.svg)](https://github.com/oharastream/ohara/releases)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![jFrog Bintray](https://img.shields.io/bintray/v/oharastream/ohara/ohara-client.svg)](https://bintray.com/oharastream/ohara)
 
-----------
+# Ohara
+
+Ohara, which is a big data platform powered by [Apache Kafka](https://kafka.apache.org/) and [Docker](https://www.docker.com/), 
+enables effective and quickly ways of building application at streaming and scale.
```
Comment (konekoya) [inline README.md]: Should probably use _effective and quick ways..._

File: README.md [hunk 5]
```diff
@@ -1,37 +1,43 @@
...
 [![Build Status](https://builds.is-land.com.tw/buildStatus/icon?job=PostCommit-OHARA)](https://builds.is-land.com.tw/job/PostCommit-OHARA/)
 [![Documentation Status](https://readthedocs.org/projects/ohara/badge/?version=latest)](https://ohara.readthedocs.io/en/latest/?badge=latest)
+[![Github Assets](https://img.shields.io/badge/download-assets-brightgreen.svg)](https://github.com/oharastream/ohara/releases)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![jFrog Bintray](https://img.shields.io/bintray/v/oharastream/ohara/ohara-client.svg)](https://bintray.com/oharastream/ohara)
 
-----------
+# Ohara
+
+Ohara, which is a big data platform powered by [Apache Kafka](https://kafka.apache.org/) and [Docker](https://www.docker.com/),
```
Comment (konekoya) [inline README.md]: This reads better to me :)

_Ohara, a big data platform which is powered by ... _

File: README.md (hunk 3 above)
Comment (vitojeng) [reply README.md]: Excellent!

**Pull request merged**